apps/designer/app/designer/features/topbar/topbar.tsx

```diff
@@ -18,19 +18,37 @@ type TopbarProps = {
 export const Topbar = ({ config, css, project, publish }: TopbarProps) => {
   return (
     <Flex
+      className={darkTheme}
       as="header"
       align="center"
       justify="between"
       css={{
-        p: "$1",
         bc: "$loContrast",
-        borderBottom: "1px solid $slate8",
+        height: "$sizes$7",
+        "& :focus": {
```

Comment:
uhh, setting this on any focused child element? lets see what's the use case and why its necessary to override vs. not having it in the first place